multinetjs/src/index.ts

```diff
@@ -50,8 +56,8 @@ export type EdgesOptionsSpec = OffsetLimitSpec & {
   direction?: Direction;
 };
 
-export interface UploadTableOptionsSpec {
-  type: UploadType;
+export interface FileUploadOptionsSpec {
+  type: TableUploadType | GraphUploadType;
```

Comment:
This can just be

```suggestion
  type: UploadType;
```

right?

Reply:
Yep